docs/advanced-techniques.md

```diff
@@ -88,3 +88,27 @@ A module is a standalone configuration object that can reference client code, re
 A merged module's code is treated the same as the application's code. The resulting application's bundles include the module code and configurations as if they were always a part of the application.
 
 For an example on using modules, see the [Hacker News](https://github.com/reimagined/resolve/tree/master/examples/hacker-news) sample application. This application makes use of the authentication module and the comments module provided by reSolve.
```

Comment:
"on using modules" Do you mean 'on how to use modules' or 'how modules are used'?

Reply:
"makes use of the" So you mean it 'uses'?

Reply:
"the comments module provided by reSolve" Does this need to be in passive voice?

Reply:
> "on using modules" Do you mean 'on how to use modules' or 'how modules are used'?

Changed to "on how to use modules"

Reply:
> "makes use of the" So you mean it 'uses'?

Yes, changed to "uses"

Reply:
Changed to "the comments module that reSolve provides"